src/components/pages/dashboard-pages/Notifications.jsx

```diff
@@ -22,9 +23,11 @@ const Notifications = ({ socket }) => {
   const [applicants, setApplicants] = useState([]);
   const [classesError, setClassesError] = useState(null);
   const [applicantsError, setApplicantsError] = useState(null);
```

Comment:
I do not see where we use applicantsError

Reply:
Thank you, Anna!
Done